src/main/java/common/CommonActions.java

```diff
@@ -1,73 +1,119 @@
+				Loggers.logTheTest(input + " <-----> has been put into <-----> " + element);
+			} catch (NoSuchElementException | NullPointerException e) {
+				Loggers.logTheTest(element + "<----------> has not been found becuase of ...\n" + e.getMessage() );
+			}		
+		}
+		
+		public static void elementDisplayed(WebElement element) {
+			try {
+				boolean flag = element.isDisplayed();
+				Loggers.logTheTest(element + "<---------> is Displayed, " + flag);
+			} catch (NoSuchElementException | NullPointerException e) {
+				Loggers.logTheTest(element + "<----------> is not Displayed\n" + e.getMessage() );
+			}		
+		}
+		
+		public static void elementEnabled(WebElement element) {
+			try {
+				boolean flag = element.isEnabled();
+				Loggers.logTheTest(element + "<---------> is Enabled, " + flag);
+			} catch (NoSuchElementException | NullPointerException e) {
+				Loggers.logTheTest(element + "<----------> is not Displayed\n" + e.getMessage() );
+			}
+		}
+		
+		public static void elementSelected (WebElement element){
+			try {
+				boolean flag = element.isSelected();
+				Loggers.logTheTest(element + "<---------> is Selected, " + flag);
+			} catch (NoSuchElementException | NullPointerException e) {
+				Loggers.logTheTest(element + "<----------> is not Displayed\n" + e.getMessage() );
+			}
+		}
+		
+		public static void verifyTitle(WebDriver driver, String expectedTitle) {
+			try {
+				String actualTitle = driver.getTitle();
+				Loggers.logTheTest("Actual Title is : " + actualTitle + " ---> And Expected Title is :" + expectedTitle);
+				Assert.assertEquals(actualTitle, expectedTitle, "Title doesn't match up");
+			} catch (NullPointerException e) {
+				Loggers.logTheTest("Driver is not initiated properly Or Title doesn't match up");
+				Assert.fail();
+			}
+			
+		}
+		
+		public static void verifyCurrentUrl(WebDriver driver, String expectedURL) {
+			try {
+				String currentURL = driver.getCurrentUrl();
+				Loggers.logTheTest("Current URL : " + currentURL + ", Expected URL : " + expectedURL);
+				Assert.assertEquals(currentURL, expectedURL, "Current URL is not correct");	
+			} catch (NullPointerException e) {
+				Loggers.logTheTest("Driver is not initiated properly Or Current URL doesn't match");
+				Assert.fail();
+			}
+				
+		}
+		
+		public static void verifyTextOfTheWebElement(WebElement element, String expected) {
+			try {
+				String actual = element.getText();
+				Loggers.logTheTest(element + " ---> Actual text : " + actual + ". Expected text : " + expected);
+				Assert.assertEquals(actual, expected, "Text In the WebElement doesn't match");
+			} catch (NoSuchElementException | NullPointerException e) {
+				Loggers.logTheTest(element + "<----------> is not Displayed or Text doesn't match\n" + e.getMessage() );
+			}
+		}
+		
+		public static void clearTextFromTheField(WebElement element) {
+			try {
+				element.clear();
+				Loggers.logTheTest("The Text from the " + element + " ---> is cleared");
```

Comment:
Can you use log4j instead of java util logger?

Reply:
We are using Java util Logger for now, we will work on it after discussing with team